src/styles/header.scss

```
@@ -0,0 +1,17 @@
+@import "src/styles/variables";
+
+.header {
+  display: flex;
+  margin: 0 auto 50px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  margin: auto 50px;
+}
```

```suggestion
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
  
  &__logo {
  margin: auto 50px;
}
}
```
Используй вложенность